Hush/Assets/Scripts/Doors/Door.cs

```diff
@@ -57,10 +63,12 @@ private void OpenDoor(InputAction.CallbackContext context)
             if (!CanOpenDoor)
                 return;
 
-            // TODO Door opening animation
-            wall.SetActive(false);
-            
-            // respawn back into scene
+            anim.SetTrigger("Open");
+            Invoke("LoadNext", 2f);
```

Comment:
```suggestion
            Invoke(nameof(LoadNext), 2f);
```

Reply:
Changed in 9454f5848f7592ea000731146dfcc48c658a318a